backend/utils/minify.ts

```diff
@@ -0,0 +1,76 @@
+const minify = (json: string) => {
```

Comment:
what does this code do?

Reply:
It minifies the settings.json and removes all the comments that JSON.parse would not understand.

Reply:
add just small JSDoc on top like

```js
/*
  * It's allow us to remove comment on json
  */
```

So on ide when we hover function we know what she doe